Cleanup Concepts page (#2409)
flyteorg/flyte: rsts/concepts/console.rst

.. _divedeep-console:

#############
FlyteConsole
#############

> This is the web UI for the Flyte platform. The results of running FlyteConsole are displayed in this graph, explained below:

Reviewer: ```suggestion
FlyteConsole is the web UI for the Flyte platform. Here's a video that dives into the graph UX:
```

Storybook
==========

> This project supports `Storybook <https://storybook.js.org/>`_.
> Component stories live next to the components they test, in a ``__stories__``
> directory, with the filename pattern ``{Component}.stories.tsx``.

Reviewer: ```suggestion
FlyteConsole uses `Storybook <https://storybook.js.org/>`__.
Component stories live next to the components they test in the ``__stories__``
directory with the filename pattern ``{Component}.stories.tsx``.
```
